packages/core/src/Stores/client-store.js

```diff
@@ -1138,6 +1178,11 @@ export default class ClientStore extends BaseStore {
                 this.root_store.gtm.pushDataLayer({
                     event: 'signup',
                 });
+
+                // Show Accounts popup to European users
```

Comment:
Can we refactor this to its own action?

Reply:
@reza I didn't get your point! you mean extract this if condition and create a new action for it?

Reply:
@reza I changed the if condition to: `this.root_store.ui.toggleAccountTypesModal(this.is_europe);`